.github/workflows/github.yml

```diff
@@ -384,3 +384,28 @@ jobs:
         name: ${{ env.VCMI_PACKAGE_FILE_NAME }}
         path: |
           ${{ env.ANDROID_APK_PATH }}
+
+
+  deploy-src:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: pwsh
+    steps:
+        - uses: actions/checkout@v4
+          with:
+            submodules: recursive
+            
+        - name: Create source code archive (including submodules)
+          run: |
+            git archive HEAD -o "release.zip" --worktree-attributes -v
+            git submodule update --init --recursive
+            git submodule --quiet foreach 'cd "$toplevel"; zip -ru "release.zip" "$sm_path"'
+            unzip -q release.zip -d ./src
```

Comment:
Probably not needed?
Yes, archive in archive is weird, but first step when preparing release is "unpack all artifacts", so having uniform format is more convenient.

Reply:
Ah, didn't see that part. I think it's good not to unzip it here then :+1:

Reply:
It is something I do locally on my system, so yeah, you won't find it in our CI scripts.